apis/compute/v1beta1/subnetwork_types.go

```diff
@@ -69,8 +41,11 @@ type SubnetworkParameters struct {
 	// +immutable
 	Network *string `json:"network,omitempty"`
 
-	// NetworkRef references to a Network and retrieves its URI
-	NetworkRef *NetworkURIReferencerForSubnetwork `json:"networkRef,omitempty" resource:"attributereferencer"`
+	// NetworkRef references a Network and retrieves its URI
+	NetworkRef *runtimev1alpha1.Reference `json:"networkRef,omitempty"`
+
+	// NetworkSelector selects a reference to a Network
+	NetworkSelector *runtimev1alpha1.Selector `json:"networkSelector,omitempty"`
```

Comment:
Any reason for not including `optional` and `immutable` tags here?

Reply:
Hrm, my process for making these new selector fields involved copying, pasting, then editing the equivalent ref fields and it looks like these refs were also missing the tags. Will add them.